README.md

```diff
@@ -47,20 +56,85 @@ and ask!
+is not emerging on its own, the approver may also step in as a
+mediator. The approver do not need to be a subject-matter expert for
+the subject of the design, although it can help if they are.
+
+Choosing the appropriate approver depends on the scope of an
+enhancement. If it is limited in scope to a given team or component,
+then a peer or lead on that team or pillar is appropriate.  If an
+enhancement captures something more broad in scope, then a member of
+the OpenShift staff engineers team or someone they delegate would be
+appropriate.  Examples of broad scope are proposals that change the
+definition of OpenShift in some way, add a new required dependency, or
+change the way customers are supported.  Use your best judgement to
+determine the level of approval needed.  If you’re not sure, ask a
+staff engineer to help find a good approver by posting in
+`#forum-arch` on the CoreOS Slack server and tagging
+`@aos-staff-engineers`.
+
+The set of reviewers for an enhancement proposal can be anyone that
+has an interest in this work or the expertise to provide a useful
+input/assessment.  At a minimum the reviewers should include the lead
+(or a delegate) for any team that will need to do work for this EP, or
+whose team will own/support the resulting implementation.  Please
+indicate what aspect of the EP you expect them to be concerned with so
+they do not always need to review the entire EP in depth.
+
+## How Can an Author Help Speed Up the Review Process?
+
+Enhancements should have agreement from all stakeholders prior to
+being approved and merged. Reviews are not time-boxed (see Life-cycle
+below). We manage the rate of churn in OKD by asking component
+maintainers to act as reviewers in addition to everything else that
+they do.  If it is not possible to attract the attention of enough of
+the right maintainers to act as reviewers, that is a signal that the
+project's rate of change is maxed out. With that said, there are a few
+things that authors can do to help keep the conversation moving along.
+
+1. Respond to comments quickly, so that a reviewer can tell you are
+   engaged.
+2. Push update patches, rather than force-pushing a replacement, to
+   make it easier for reviewers to see what you have changed. Use
+   descriptive commit messages on those updates, or plan to use
+   `/label tide/merge-method-squash` to have them squashed when the
+   pull request merges.
+3. Do not rely solely on the enhancement for visibility of the
+   proposal. For high priority work, or if the conversation stalls
+   out, you can start a thread in `#forum-arch` on the CoreOS Slack
+   server or bring the enhancement to one of the weekly architecture
+   review meetings for discussion. If you aren't sure which meeting to
+   use, work with a staff engineer to find a good fit.
+4. Be mindful of the fact that reviewers have work other than reading
+   updates to your enhancement. Avoid overusing pings on Slack to get
+   their attention.
```

Comment:
this last item isn't a thing you can do to keep the conversation moving along, it's a thing not to do.  So i wouldn't include it in this list.

if you do want to include advice about slack pings in the list, i'd reframe it as something like:

4. If the conversation otherwise seems stuck, slack pings can be useful tool to prod someone, but it's generally appropriate to give people at least a business day or two to respond in the github thread first, before reaching out to them directly on slack, so that people can manage their work queue/disruptions.

Reply:
See 0e799c9